pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2629,4 +2629,6 @@ self: super: {
   llvm-ffi = super.llvm-ffi.override {
     LLVM = pkgs.llvmPackages_13.libllvm;
   };
+
+  libfuse3 = dontCheck super.libfuse3;
```

Comment:
Could you add a comment here about why `dontCheck` is needed?

Reply:
done